os/include/arpa/inet.h

```diff
@@ -180,6 +186,9 @@ uint16_t ntohs(uint16_t ns);
 /**
  * @brief Convert the unsigned integer hostlong from host byte order to network byte order
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised